src/ethereum_metrics_exporter/ethereum_metrics_exporter_launcher.star

```diff
@@ -0,0 +1,49 @@
+shared_utils = import_module("../shared_utils/shared_utils.star")
+constants = import_module("../package_io/constants.star")
+static_files = import_module("../static_files/static_files.star")
+ethereum_metrics_exporter_context = import_module("../ethereum_metrics_exporter/ethereum_metrics_exporter_context.star")
+
+HTTP_PORT_ID = "http"
+STARTING_HTTP_PORT_NUMBER = 18000
+
+def launch(
+    plan,
+    index,
+    pair_name,
+    ethereum_metrics_exporter_service_name,
+    ethereum_metrics_exporter_image,
+    el_client_context,
+    cl_client_context,
+):
+    listening_port = STARTING_HTTP_PORT_NUMBER + index
```

Comment:
There is no need to do this. Kurtosis takes care that each service gets its own independent randomized port. So you can safely assign the same port to multiple services.

Reply:
Fixed! Tyvm 🔥